internal/testdata/addons-imageset/reference-addon/metadata/stage/addon.yaml

```diff
@@ -0,0 +1,25 @@
+label: api.openshift.com/addon-reference-addon
+enabled: true
+addonOwner: MT-SRE Team <sd-mt-sre@redhat.com>
+quayRepo: quay.io/osd-addons/reference-addon
+testHarness: quay.io/miwilson/addon-samples
+installMode: OwnNamespace
+targetNamespace: redhat-reference-addon
+namespaces:
+  - redhat-reference-addon
+ocmQuotaName: addon-reference-addon
+ocmQuotaCost: 1
+operatorName: reference-addon
+channels:
+  - currentCSV: reference-addon.v0.0.1
+    name: alpha
+defaultChannel: alpha
+bundleParameters: {}
+namespaceLabels: {}
+namespaceAnnotations: {}
+addonImageSetVersion: v0.0.1
```

Comment:
Just a heads up, the managed-tenants-cli python package expects `addonImageSetVersion` strings without the prefix "v". Ie, `0.01` instead of `v0.0.1`

Reply:
fixed